api/get.mjs

```diff
@@ -0,0 +1,68 @@
+import Grubhub from "../services/Grubhub.mjs";
+import { HTTPResponseError } from "../errors/http.mjs";
+
+// store any response cookies to be sent back after search
+// currently this is used to update Postamtes search cookies
+let resCookies = [];
+
+/* Search grubhub
+ * @param {string} query the query to search from (ex:pizza, mcdonalds)
+ * @return {object} either a stores object or error
+ */
+const searchGrubhub = async (searchData) => {
```

Comment:
This method already exists in `/api/search` so instead of pasting the same code, export it from the search api and import it in the get api.